openedx/features/course_experience/course_tools.py

```diff
@@ -57,6 +69,16 @@ def url(cls, course_key):
         """
         raise NotImplementedError("Must specify a url for a course tool.")
 
+    @classmethod
+    def data(cls):
+        """
+        Additional data to send with a form submission
+        """
+        if cls.http_method == HttpMethod.POST:
+            return {}
+        else:
+            return None
```

Comment:
What's the difference between None and {} for this?

Reply:
I think in practice, there wouldn't be a difference. I did it more for the semantic difference of a POST having data and a GET having no data. Thoughts?

Reply:
That's fine.